WalletWasabi.Fluent/Models/UiTriggers.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Reactive;
+using System.Reactive.Linq;
+using ReactiveUI;
+using WalletWasabi.Blockchain.TransactionProcessing;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Wallets;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Models;
+
+public class UiTriggers
+{
+	private readonly WalletViewModel _walletViewModel;
+	private readonly Wallet _wallet;
+
+	public UiTriggers(WalletViewModel walletViewModel)
+	{
+		_walletViewModel = walletViewModel;
+		_wallet = _walletViewModel.Wallet;
+	}
+
+	/// <summary>
+	/// Triggers on subscription and when a transaction to the wallet is processed.
+	/// </summary>
+	public IObservable<Unit> WalletRelevantTransactionProcessed =>
+		Observable.FromEventPattern(_wallet.TransactionProcessor, nameof(TransactionProcessor.WalletRelevantTransactionProcessed)).ToSignal().StartWith(Unit.Default);
```

Comment:
Oops! This is lying because it says `WalletRelevantTransactionProcessed` but it will emit a signal when subscribed :)
I know you do this to force an initial update, but I think it will be better if we move that responsibility to the consumer of the sequence (the one that is interested in the initial signal).

Reply:
All the triggers give a sign on subscription in this class so the developers would expect that this one also does that. Also the initial sign is needed everywhere and the point of this class is to give ready to use objects.

You are right about he naming btw, I will change it.